### DISCUSSION TITLE
使用Zygisk_mod而不是Zygisk Next

### DISCUSSION THREAD

**Discussion by Keeperorowner** [bmax121/APatch#760] [Ideas]
好处不仅仅是开源，webUI也更好看，开发者也为APatch做了专门的适配

**Comment by EvenLRs** [comment]
你说的是Admirepowered/Zygisk_mod这个项目？我看这个项目已经归档了，后续还有人接手维护吗？

**Comment by Keeperorowner** [comment]
> 你说的是Admirepowered/Zygisk_mod这个项目？我看这个项目已经归档了，后续还有人接手维护吗？

Rezygisk也是一个选择

https://github.com/PerformanC/ReZygisk/releases/tag/v1.0.0-rc.1
